stdlib/public/core/FloatingPoint.swift.gyb

```diff
@@ -28,7 +28,7 @@ word_bits = int(CMAKE_SIZEOF_VOID_P) * 8
 /// additional operations that only make sense for a fixed radix, and also
 /// provides default implementations of some of the FloatingPoint APIs.
 public protocol FloatingPoint: Comparable, IntegerLiteralConvertible,
-  SignedNumber, AbsoluteValuable {
+  SignedNumber, AbsoluteValuable, Strideable {
```

Comment:
I thought there was some discussion about removing Strideable from floating point types.  Was that a separate proposal?